include/deal.II/base/quadrature_lib.h

```diff
@@ -439,6 +439,71 @@ class QSorted : public Quadrature<dim>
                   const std::pair<double, Point<dim> > &b);
 };
 
+/**
+ * Telles quadrature of arbitrary order.
+ *
+ * The coefficients of these quadrature rules are computed using
+ * a non linear variable change starting from a Gauss-Legendre
+ * quadrature formula.
+ * This is done using a cubic polynomial,
+ * n = a x^3 + b x^2 + c x + d
```

Comment:
Can you set formulas in math mode please? (Follows latex syntax), $$ for inline formulas,